master/docs/examples/hello.cfg

```diff
@@ -1,72 +1,71 @@
 # -*- python -*-
 # ex: set syntax=python:
 
-from buildbot.plugins import *
+from buildbot.plugins import buildslave
+from buildbot.plugins import changes
+from buildbot.plugins import util
+from buildbot.plugins import steps
```

Comment:
well the whole point was to just have everything in the global namespace, so that people can just copy paste samples from the doc, and not worry about what to import.